example-get-started-experiments/code/data/.gitignore

```diff
@@ -1,3 +1,2 @@
 /pool_data
-/test_data
-/train_data
+/yolo_dataset
```

Comment:
does it need to be prefixed with `yolo_` - is that dataset yolo specific?

Reply:
is not needed. just to give a name that indicates the format